Agreements/Internal/28_Trent.txt

```diff
@@ -3,25 +3,23 @@ COPX:*:*:EVSON:EGCN:*:London S28:Trent:*:20000:EVSON25
 
 COPX:*:*:DENBY:EGNM:*:London S28:Trent:*:20000:DENBY40
 COPX:*:*:DENBY:EGNH:*:London S28:Trent:*:20000:DENBY40
+COPX:*:*:DENBY:EGNO:*:London S28:Trent:*:20000:DENBY40
 
-COPX:*:*:ELVOS:EGCC:*:London S28:Trent:*:20000:ELVOS
-COPX:*:*:ELVOS:EGGP:*:London S28:Trent:*:20000:ELVOS
-COPX:*:*:ELVOS:EGNR:*:London S28:Trent:*:20000:ELVOS
+COPX:*:*:TNT:EGNM:*:London S28:Trent:*:20000:TNT10
+COPX:*:*:TNT:EGNH:*:London S28:Trent:*:20000:TNT10
+COPX:*:*:TNT:EGNO:*:London S28:Trent:*:20000:TNT10
 
-COPX:*:*:EMBOR:EGCC:*:London S28:Trent:*:20000:EMBOR10
-COPX:*:*:EMBOR:EGGP:*:London S28:Trent:*:20000:EMBOR10
-COPX:*:*:EMBOR:EGNR:*:London S28:Trent:*:20000:EMBOR10
+COPX:*:*:TNT:EGCC:*:London S28:Trent:*:20000:TNT25
+COPX:*:*:TNT:EGGP:*:London S28:Trent:*:20000:TNT25
+COPX:*:*:TNT:EGNR:*:London S28:Trent:*:20000:TNT25
 
-COPX:*:*:*:EGCC:*:London S28:Trent:*:20000:TNT25
-COPX:*:*:*:EGGP:*:London S28:Trent:*:20000:TNT25
-COPX:*:*:*:EGNR:*:London S28:Trent:*:20000:TNT25
+COPX:*:*:EMBOR:EGNM:*:London S28:Trent:*:20000:EMBOR
+COPX:*:*:EMBOR:EGNH:*:London S28:Trent:*:20000:EMBOR
+COPX:*:*:EMBOR:EGNO:*:London S28:Trent:*:20000:EMBOR
 
-COPX:*:*:DENBY:EGNO:*:London S28:Trent:*:20000:DENBY55
-COPX:*:*:DENBY:EGNJ:*:London S28:Trent:*:20000:DENBY55
-COPX:*:*:DENBY:EGCN:*:London S28:Trent:*:20000:DENBY55
+COPX:*:*:ELVOS:EGCC:*:London S28:Trent:*:20000:ELVOS
+COPX:*:*:ELVOS:EGGP:*:London S28:Trent:*:20000:ELVOS
+COPX:*:*:ELVOS:EGNR:*:London S28:Trent:*:20000:ELVOS
```

Comment:
ELVOS should come before TNT, as all ELVOS inbounds will route via TNT - so this agreement should come first.